css/css-color/oklab-l-almost-0.html

```diff
@@ -0,0 +1,18 @@
+<!DOCTYPE html>
+<meta charset="utf-8">
+<title>CSS Color 4: OKLab and OKLCH</title>
+<link rel="help" href="https://drafts.csswg.org/css-color-4/#specifying-oklab-oklch">
+<link rel="match" href="oklab-l-almost-0-ref.html">
+<meta name="assert" content="oklab() with lightness very close to 0 should render the same as 0">
+<style>
+    .test { background-color: red; width: 12em; height: 6em; }
+    /* Non-zero a/b is used to show if the result is black or not, but the
+     * test passes as long as it's the same color. */
+    .ref { background-color: oklab(0 0.15 0.15); width: 12em; height: 6em; }
+    .test { background-color: oklab(0.0001% 0.15 0.15); }
+</style>
+<body>
+    <p>Test passes if you see a single color.</p>
```

Comment:
Perhaps the "Test passes if you see a single color." wording could be a little clearer about when you're seeing the success/failure cases?  (Maybe it should mention a *square* of a single color?)

Reply:
This bugged me too, and an earlier iteration added a border and said something like a single color inside the box. But that required adding a wrapping div for the border.


Any thoughts on how to make it clearer without an extra div? Or maybe I should just not make the test as minimal as possible?